drivers/clock_control/nrf_clock_calibration.c

```diff
@@ -231,7 +256,8 @@ void z_nrf_clock_calibration_init(struct device *dev)
 	nrf_clock_event_clear(NRF_CLOCK, NRF_CLOCK_EVENT_DONE);
 	nrf_clock_int_enable(NRF_CLOCK, NRF_CLOCK_INT_DONE_MASK);
 
-	clk_dev = dev;
+	hf_mgr = z_nrf_clock_control_get_onoff(CLOCK_CONTROL_NRF_SUBSYS_HF);
+	lf_mgr = z_nrf_clock_control_get_onoff(CLOCK_CONTROL_NRF_SUBSYS_LF);
```

Comment:
The `dev` parameter is no longer used. Instead of it, you could pass to `z_nrf_clock_calibration_init()` the pointer to the onoff_manager array, and actually store only this one pointer.